src/components/Table/readme.md

```diff
@@ -577,51 +304,81 @@ const StatusBadge = ({ value }) => <Badge label={value} variant="lightest" style
 </div>
 ```
 
+##### Table with dynamic row actions
+
+```js
+import React from 'react';
+import { Table, Column, ButtonMenu, ButtonGroup, ButtonIcon, Badge, MenuItem } from 'react-rainbow-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCog, faEllipsisV } from '@fortawesome/free-solid-svg-icons';
+import { dynamicActionData } from './exampleData.js'
+
+
+const StatusBadge = ({ value }) => {
+    if (value === 'verified') {
+        return <Badge label={value} variant="outline-brand" />;
+    }
+    return <Badge label={value} variant="lightest" />;
+};
+
+const MenuAction = ({value,name}) => {
+    if(value === 'verified'){
+        return  <MenuItem label="Delete" />
+    }
+    return (
+        <>
+            <MenuItem label="Delete" onClick={() => console.log(`Delete ${name}`)}/>
+            <MenuItem label="Edit" onClick={() => console.log(`Edit ${name}`)}/>
+        </>
+    );
+};
+
+const ButtonAction = (props) => {
+    const {value, row:{name}}=props;
+    return (
+        <ButtonMenu
+            id="button-menu-2"
```

Comment:
take care this id is not used already in other Table example

Reply:
I checked and the id is not used in any other example